mdx-models/src/main/java/com/mx/path/model/mdx/model/profile/NewPassword.java

```diff
@@ -0,0 +1,44 @@
+package com.mx.path.model.mdx.model.profile;
+
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+import com.mx.path.model.mdx.model.MdxBase;
+import com.mx.path.model.mdx.model.challenges.Challenge;
+
+public final class NewPassword extends MdxBase<NewPassword> {
```

Comment:
Let's add a javadoc block up here explaining that this is replacing the old one.

Reply:
Done.